cds/modules/deposit/api.py

```diff
@@ -735,9 +735,14 @@ def project(self):
                 project_id = self['_project_id']
             except KeyError:
                 return None
-            project_pid = PersistentIdentifier.query.filter_by(
-                pid_value=project_id).one()
-            self._project = Project.get_record(id_=project_pid.object_uuid)
+            try:
+                # get the deposit project
+                self._project = deposit_project_resolver(project_id=project_id)
+            except PIDDoesNotExistError:
+                # get the record project
+                _, record = record_resolver.resolve(project_id)
+                project_id = record['_deposit']['id']
+                self._project = deposit_project_resolver(project_id=project_id)
```

Comment:
yes, it can. 
But in that case is ok that is not catch.
This should be a use case not usual, but I think can happen:
  - everything is published
  - edit a video only
  - access somehow the (fake) property `video.project`
In this case `self['_project_id']` is pointing to the project record but we should return the project deposit.
If it doesn't exist it means that something is wrong :)